app/controllers/list_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def show
+    @list = List.find_by_id(params[:id])
+    render status: @list ? :ok : :no_content
+  end
+
+  def create
+    @list = List.new(list_params.merge(therapist: current_user.role))
+    status = @list.save ? :ok : :bad_request
+    render json: {list: @list}, status: status
```

Comment:
rather than saving the status and rendering the json either way, it might be a better idea to render two different things depending on whether or not the save works, like:
```
if @list.save
  render json: {list: @list}, status: :ok
else
  render json: {errors: @list.errors.full_messages}, status: :bad_request
end
```